code/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -25,13 +25,13 @@
 	var/list/trophies = list()
 	var/charged = TRUE
 	var/charge_time = 15
-	var/detonation_damage = 50
-	var/backstab_bonus = 30
+	var/detonation_damage = 25
+	var/backstab_bonus = 15
 
 /obj/item/kinetic_crusher/ComponentInitialize()
 	. = ..()
 	AddComponent(/datum/component/butchering, 60, 110) //technically it's huge and bulky, but this provides an incentive to use it
-	AddComponent(/datum/component/two_handed, force_unwielded=0, force_wielded=20)
+	AddComponent(/datum/component/two_handed, force_unwielded=0, force_wielded=10)
```

Comment:
I think this is too bad, considering a toolbox does 12 if I'm not mistaken. Perhaps bringing it up to 14 (or any value higher than the knife) could be a good change, especially considering how much worse melee is against simplemobs